frontend/dockerfile/linter/ruleset.go

```diff
@@ -29,18 +29,10 @@ var (
 			return "Empty continuation line"
 		},
 	}
-	RuleConsistentInstructionCasing = LinterRule[func(string) string]{
+	RuleConsistentInstructionCasing = LinterRule[func(string, string) string]{
 		Name:        "ConsistentInstructionCasing",
-		Description: "Instructions should be in consistent casing (all lower or all upper)",
-		URL:         "https://docs.docker.com/go/dockerfile/rule/consistent-instruction-casing/",
-		Format: func(command string) string {
-			return fmt.Sprintf("Command '%s' should be consistently cased", command)
-		},
-	}
-	RuleFileConsistentCommandCasing = LinterRule[func(string, string) string]{
```

Comment:
Maybe not needed yet, but generally I think we need to keep old rules/URLs in here so the docs are still generated for them. Otherwise old client will hit issue, click on a link and get 404

Reply:
That's a good thought to bring up, and you're definitely right. There may be a case for adding some sort of `Deprecated` flag to the ruleset so that documentation can make it as such.